### ISSUE TITLE
The speed of linear measurement on the image is very slow

### ISSUE THREAD

**Issue by JiangWeian** [OHIF/Viewers#2502]
The test image:
[IMAGES.zip](https://github.com/OHIF/Viewers/files/6929159/IMAGES.zip)

linear measurement:
![TIM图片20210804105910](https://user-images.githubusercontent.com/56569509/128115216-3e9a3fd7-17cc-4143-af68-1885288cb263.png)

**Comment by JiangWeian** [comment]
Sorry, there is no problem when there are only 3 images. There are 1200 images in this complete examination

**Comment by JiangWeian** [comment]
[test.zip](https://github.com/OHIF/Viewers/files/6929049/test.zip)
Test with these images

**Comment by JiangWeian** [comment]
There is a large image in the study, such as 60M, which will cause jams such as sequence switching and line measurement

**Comment by JiangWeian** [comment]
@swederik, @dannyrb, It took 2 seconds from the componentdidupdate() method to the render() method of the OHIFCornerstoneViewport class:
![TIM图片20210805102855](https://user-images.githubusercontent.com/56569509/128281996-3adbecaf-0a5c-45f5-b433-5a890d4049da.png)

**Comment by JiangWeian** [comment]
@swederik, @dannyrb, This callback is to execute cornerstone.renderToCanvas():
![TIM图片20210806082300](https://user-images.githubusercontent.com/56569509/128437773-d4887caa-eed1-4fa4-9a73-7ff3e49355cd.png)

**Comment by JiangWeian** [comment]
I have found the reason. The thumbnail on the left is often rendered. If it contains large images (such as 60M), it will slow down the switching sequence, line measurement and other operations.
I modified the useeffect() of imagethumbnail.js:
useEffect(() => {
// The left thumbnail does not need to be refreshed repeatedly if the image has been loaded
if(! isLoading)
    return;

if (image.imageId) {
    cornerstone.renderToCanvas(canvasRef.current, image);
    setLoading(false);
}
}, [canvasRef, image, image.imageId]);
